### DISCUSSION TITLE
My applications are not approved

### DISCUSSION THREAD

**Discussion by wahbisaied** [community/community#144464] [GitHub Education]
### Hi everyone,

Hi everyone, 
I applied 4 times to activate my github education account but everytime i receive a rejection after some minutes, 
+ first 3 applications with my student IDcard ( written in french & arabic ) saying ( 
Suggested proof to expedite your approval:
Please use your device camera to submit academic affiliation documents. ) 
+ 4th trial with official enrollment document ( in arabic ) getting this rejection:
----------------------------------------------
Suggested proof to expedite your approval:

Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.

Please use your device camera to submit academic affiliation documents.

Please ensure that your academic affiliation document contains your first name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). Variations of your name and nicknames are unacceptable.

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.
---------------------------------------------------------

The ID card and enrollment paper contain my first name and last name as written in my GitHub account. Pictures are taken with my phone camera. I activated the 2-step verification for the GitHub account and filled in my payment information. Still, I am getting rejected, and individual manual review is no longer provided on GitHub. What should I do, please?

**Comment by M-tech-cmd** [comment]
Ensure Exact Name Matching: Double-check that your GitHub profile name matches exactly with the name on your academic documents. This includes both first and last names without any variations or nicknames.

Use Clear and Legible Documents: Make sure the photos of your documents are clear and legible. Blurry or unclear images can lead to rejections. Use a high-quality camera and good lighting.

Update Profile Information: Ensure your GitHub profile is fully updated with your full name, a short bio, and any other required information. Log out and log back in after making these updates.

Submit Additional Proof: If possible, provide additional documents that clearly show your current enrollment status, such as a transcript, class schedule, or an official letter from your institution.

Contact GitHub Support: If you continue to face issues, consider reaching out to GitHub Education support directly. Explain your situation in detail and provide all relevant information. [You can contact them at support@github.com](https://github.com/orgs/community/discussions/143377)[1](https://github.com/orgs/community/discussions/143377)[2](https://github.com/orgs/community/discussions/111352).

Check GitHub Discussions: Look through GitHub community discussions for similar issues and potential solutions. [Other users might have faced and resolved similar problems](https://github.com/orgs/community/discussions/111352)[2](https://github.com/orgs/community/discussions/111352)[3](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/why-wasnt-my-application-to-github-education-for-students-approved?apiVersion=2022-11-28).
